src/search/FindInFiles.js

```diff
@@ -135,7 +135,7 @@ define(function (require, exports, module) {
             
             if (highlightEndCh <= MAX_DISPLAY_LENGTH) {
                 // Don't store more than 200 chars per line
-                line = line.substr(0, Math.min(MAX_DISPLAY_LENGTH, line.length));
```

Comment:
Thanks for noticing, it was meant to be the other way round.
I've just pushed a fix.